src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/Diagnostics/QueueClientDiagnosticsTests.cs

```diff
@@ -6,558 +6,558 @@ namespace Microsoft.Azure.ServiceBus.UnitTests.Diagnostics
     using System;
     using System.Diagnostics;
     using System.Linq;
-    using System.Threading;
     using System.Threading.Tasks;
     using Xunit;
 
     public sealed class QueueClientDiagnosticsTests : DiagnosticsTests
     {
-        private QueueClient queueClient;        
-        private bool disposed = false;
-
-        [Fact]
-        [LiveTest]
-        [DisplayTestMethodName]
-        async Task EventsAreNotFiredWhenDiagnosticsIsDisabled()
```

Comment:
Note:  This test is an example of one that is not testing diagnostics in the Service Bus client, but rather validating the filtering approach by the test utility `FakeDiagnosticsListener`.   I've relocated these to a dedicated class to try and better differentiate the context being validated.